pkg/operator/operator.go

```diff
@@ -22,6 +22,13 @@ import (
 	"net"
 	"os"
 
+	//v2
```

Comment:
I think It's easy to understand the deps here. I don't think we need these commnets